src/TableColumn/FilterModel/Generic.php

```diff
@@ -102,7 +102,8 @@ public function afterInit()
             // create a name for our filter model to save as session data.
             $this->name = 'filter_model_'.$this->lookupField->short_name;
 
-            if (@$_GET['atk_clear_filter']) {
+            $atk_clear_filter = $_GET['atk_clear_filter'] ?? false;
```

Comment:
cant this be done like 
`if(($_GET['atk_clear_filter'] ?? false)) {` ?
Would save the extra var $atk_clear_filter which doesnt seem to be used anywhere else.